CODEOWNERS

```diff
@@ -1 +1 @@
-* @brikis98 @yorinasub17
+@yorinasub17 @zackproser @rhoboat
```

Comment:
Actually, should @denis256 be added to this as well, since this is the example repo for Terragrunt?